frontend/src/pages/AboutUs/OurTeamTab.tsx

```diff
@@ -1,48 +1,281 @@
 import React from "react";
 import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
 import styles from "./AboutUsPage.module.scss";
 
-/* TODO - Align with mocks, Clean up CSS */
+const PARTNERS = [
+  {
+    imageUrl: "img/PartnerSatcher.png",
+    alt: "alt",
+  },
+  {
+    imageUrl: "img/PartnerGilead.png",
+    alt: "alt",
+  },
+  {
+    imageUrl: "img/PartnerCdc.png",
+    alt: "alt",
+  },
+  {
+    imageUrl: "img/PartnerGoogle.png",
+    alt: "alt",
+  },
+]
+
+const LEADERSHIP_TEAM = [
+  {
+    name: "Daniel Dawes, JD",
+    role: "Co-Principal Investigator",
+    imageUrl: "img/Daniel-E 1.png",
+  },
+  {
+    name: "Nelson Dunlap, JD",
+    role: "Co-Principal Investigator",
+    imageUrl: "img/DunlapNelson.png",
+  },
+  {
+    name: "Ebony Respress, MPH",
+    role: "Project Director",
+    imageUrl: "img/RespressEbony.png",
+  },
+  {
+    name: "Maisha Standifer, PhD, MPH",
+    role: "Health Policy Analyst",
+    imageUrl: "img/RespressEbony.png",
```

Comment:
Are these all supposed to be "RespressEbony.png" ?

Reply:
Just asked about this in the Slack -- will update with the actual images when I get them